lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2997,7 +3003,7 @@ PropertyWrapperInitializerInfoRequest::evaluate(Evaluator &evaluator,
         // FIXME: Record this expression somewhere so that DI can perform the
         // initialization itself.
         Expr *defaultInit = nullptr;
-        typeCheckSynthesizedWrapperInitializer(var, defaultInit);
+        typeCheckSynthesizedWrapperInitializer(var, defaultInit, dc->isLocalContext());
```

Comment:
Why is `dc->isLocalContext()` important here?

Reply:
Because visitPatternBindingDecl [will filter out local variables](https://github.com/apple/swift/blob/efc66c7af6c394a9fd297a0edf11630775e66039/lib/Sema/TypeCheckDeclPrimary.cpp#L2174), there is a need to contextualize backed property here.

Reply:
Ah sure!